install.sh

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env bash
+# modifed from https://github.com/v2fly/fhs-install-v2ray/blob/master/install-release.sh
+# You can set this variable whatever you want in shell session right before running this script by issuing:
+# export JSON_PATH='/usr/local/etc/hysteria'
+JSON_PATH=${JSON_PATH:-/etc/hysteria}
+
+curl() {
+  $(type -P curl) -L -q --retry 5 --retry-delay 10 --retry-max-time 60 "$@"
+}
+
+
+## Demo function for processing parameters
+judgment_parameters() {
+  while [[ "$#" -gt '0' ]]; do
+    case "$1" in
+      '--remove')
+        if [[ "$#" -gt '1' ]]; then
+          echo 'error: Please enter the correct parameters.'
+          exit 1
+        fi
+        REMOVE='1'
+        ;;
+      '--version')
+        VERSION="${2:?error: Please specify the correct version.}"
+        break
+        ;;
+      '-c' | '--check')
+        CHECK='1'
+        break
+        ;;
+      '-f' | '--force')
+        FORCE='1'
+        break
+        ;;
+      '-h' | '--help')
+        HELP='1'
+        break
+        ;;
+      '-l' | '--local')
+        LOCAL_INSTALL='1'
+        LOCAL_FILE="${2:?error: Please specify the correct local file.}"
```

Comment:
The `$LOCAL_FILE` is required but never used in the script, did you forget to handle the local installation?